### ISSUE TITLE
Why using different hyperparameters in VOC split2?

### ISSUE THREAD

**Issue by zccstig** [ucbdrive/few-shot-object-detection#36]
Hi authors,

I found the hyperparameters for the experiments on VOC split2 are a little bit different from the ones used in split1. For example, the learning rate in split2 is 5x of the one in split1. And the learning rate schedule (the step to decrease learning rate) is also a bit different.

My question is that are the results in the paper generated by different hyperparameters for three splits? If so, do different hyperparameters offer better results than a unified set of hyperparameters? 

Thanks for your reply in advance.

**Comment by thomasehuang** [comment]
We found that for split 2 of PASCAL VOC, a lower learning rate improves the results, although not by much. This is probably due to the different set of categories, so different hyperparameters may help.

Hope that helps!
